index.html

```diff
@@ -0,0 +1,306 @@
+
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Docker</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Automação</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Integrações de Sistemas</p>
+                </div>
+                
+            </div>
+
+            <hr>
+
+            <div id="skills-tools-block">
+
+                <h2>Sistemas e E-commerce:</h2>
+
+                <br>
+
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Oracle Netsuite, SuiteScript</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Bitrix</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Magento</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> WordPress</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Elementor</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Hospedagem</p>
+                </div>
+                
+            </div>
+
+            <hr>
+
+            <div id="skills-tools-block">
+
+                <h2>BI (Business Intelligence):</h2>
+
+                <br>
+
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Looker Studio</p>
+                </div>
+                
+            </div>
+
+            <hr>
+            
+            <!-- Botão para baixar currículo -->
+            <button class="download-cv-btn">
+                <a href="./curriculo/Currículo CV Básico Minimalista Simples Preto e Branco (1).pdf" download="Thálisson Gonçalves - Currículo">
+                    DOWNLOAD CURRÍCULO 
+                    <i class="fa-solid fa-download" style="color: #8c8c8e"></i>
+                </a>
+            </button>
+            
+        </div>
+
+        <div id="social-block">
+            <a href="https://www.linkedin.com/in/thalissongoncalves/" target="_blank"><i class="fa-brands fa-linkedin"></i></a>
+            <a href="https://github.com/thalissongoncalves" target="_blank"><i class="fa-brands fa-github"></i></a>
+            <a href="https://www.instagram.com/thalissonzpyg/" target="_blank"><i class="fa-brands fa-instagram"></i></a>
+        </div>
+        
+    </div>
+
+    
+
+    <div id="menu-block">
+        <div class="menu-btn-block">
+            <button class="menu-btn"><i class="fa-solid fa-bars fa-2xl" style="color: #8c8c8e;"></i></button>
+        </div>
+    </div>
```

Comment:
Copie os estilos do id menu-block e coloque na classe menu-btn-block. Nao precisa de 2 divs aqui.

```suggestion
    <div class="menu-btn-block">
            <button class="menu-btn"><i class="fa-solid fa-bars fa-2xl" style="color: #8c8c8e;"></i></button>
    </div>
```

Reply:
Nesse precisa pq vou colocar mais coisas em baixo do button kk